ci/license_check.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash -ie
+#Note - ensure that the -e flag is set to properly set the $? status if any command fails
+
+# Since we are using the system jruby, we need to make sure our jvm process
+# uses at least 1g of memory, If we don't do this we can get OOM issues when
+# installing gems. See https://github.com/elastic/logstash/issues/5179
```

Comment:
Is there supposed to be a heap memory setting corresponding to this comment?

Reply:
Ah, yeah, I copied and pasted the script. This script def doesn't have that concern, lemme delete that.